### PR TITLE
Fix benchmark flakiness

### PR DESCRIPTION
**Pull request by fchirica**

### CODE REVIEW

**Comment by arvidn** [comment]
https://github.com/Chia-Network/chia-blockchain/actions/runs/6563702905/job/17828538516?pr=16649#step:13:1314
```
=================================== FAILURES ===================================
____________ test_benchmark_batch_insert_speed[pre=1000,count=1000] ____________
tests/core/data_layer/test_data_store.py:1431: in test_benchmark_batch_insert_speed
    with benchmark_runner.assert_runtime(seconds=case.limit):
E   AssertionError: 26.133334149999996 seconds not less than 24 seconds ( 109 % )
=========================== short test summary info ============================
FAILED tests/core/data_layer/test_data_store.py::test_benchmark_batch_insert_speed[pre=1000,count=1000] - AssertionError: 26.133334149999996 seconds not less than 24 seconds ( 109 % )
```

File: .github/workflows/benchmarks.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
     runs-on: benchmark
     container:
       image: ubuntu:22.04
-    timeout-minutes: 30
+    timeout-minutes: 60
```
Comment (arvidn) [inline .github/workflows/benchmarks.yml:31]: is this really necessary?
I thought the benchmarks typically only takes about 10 minutes

File: tests/core/full_node/test_performance.py [hunk 2]
```diff
@@ -119,7 +119,7 @@ async def test_full_block_performance(
         pr = cProfile.Profile()
         pr.enable()
```
Comment (arvidn) [inline tests/core/full_node/test_performance.py:120]: I would think the profilers slow this down a bit. We should probably remove them.

File: .github/workflows/benchmarks.yml (hunk 1 above)
Comment (altendky) [reply .github/workflows/benchmarks.yml:31]: The changes this week dropped 'normal' runtimes from 28 minutes to 9.

Comment (fchirica) [reply .github/workflows/benchmarks.yml:31]: ok, given this i'll reduce to 30 again.

**Review by altendky — APPROVED** [review]

**Review by emlowe — APPROVED** [review]

**Pull request merged**